docs/intro/tutorial.rst

```diff
@@ -322,35 +322,42 @@ there could become a very tedious task. To make it easier, you can
 use Firefox Developer Tools or some Firefox extensions like Firebug. For more
 information see :ref:`topics-firebug` and :ref:`topics-firefox`.
 
-After inspecting the page source, you'll find that the web site's information
-is inside a ``<ul>`` element, in fact the *second* ``<ul>`` element.
+After inspecting the page source, you'll find that every quote in the website
+is inside a separate ``<div class="quote">`` element, such as::
 
-So we can select each ``<li>`` element belonging to the site's list with this
-code::
+    <div class="quote">
+        <span class="text">“We accept the love we think we deserve.”</span>
+        <span>by <small class="author">Stephen Chbosky</small></span>
+        <div class="tags">
+            Tags:
+            <meta class="keywords"> 
+            <a class="tag" href="/tag/inspirational/page/1/">inspirational</a>
+            <a class="tag" href="/tag/love/page/1/">love</a>
+        </div>
+    </div>
 
-    response.xpath('//ul/li')
 
-And from them, the site's descriptions::
+So we can select each ``<div class="quote">`` element belonging to the site's 
+list with this code::
 
-    response.xpath('//ul/li/text()').extract()
+    response.xpath('//div[@class="quote"]')
 
-The site's titles::
+From each quote block, we can select text with::
 
-    response.xpath('//ul/li/a/text()').extract()
+    response.xpath('//div[@class="quote"]/span[@class="text"]/text()').extract()
```

Comment:
It's a bit odd to say "From each code block" and then use `response.xpath()`, because it returns all quotes extracted in a list.
I realize this will be rewritten later, but to prevent from getting inconsistencies into master, we could use text like `To get the text for each quote:`